src/Microsoft.ApplicationInsights.AspNet/TelemetryInitializers/ComponentVersionTelemetryInitializer.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet.TelemetryInitializers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Channel;
+    using DataContracts;
+    using Microsoft.AspNet.Http;
+    using Microsoft.Extensions.Configuration;
+
+    /// <summary>
+    /// A telemetry initializer that populates telemetry.Context.Component.Version to the value read from project.json
+    /// </summary>
+    public class ComponentVersionTelemetryInitializer : TelemetryInitializerBase
+    {
+        private const string _versionConfigurationOption = "dependencies:Microsoft.ApplicationInsights.AspNet";
```

Comment:
There is no "version" entry on the project.json file of the test project

\test\Microsoft.ApplicationInsights.AspNet.Tests\project.json

I can assume there will always be one or return null.